.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -22,7 +22,7 @@ call %INSTALLER_DIR%\install_miniconda3.bat
 
 
 :: Install ninja and other deps
-if "%REBUILD%"=="" ( pip install -q "ninja==1.10.0.post1" dataclasses typing_extensions "expecttest==0.1.3" )
+if "%REBUILD%"=="" ( pip install -q "ninja==1.10.0.post1" dataclasses typing_extensions "expecttest==0.1.3" packaging )
```

Comment:
Needed for https://github.com/pytorch/pytorch/blob/ded4ca5f152f237d3b91df8d0412508dda79fe4d/.jenkins/pytorch/win-test-helpers/build_pytorch.bat#L129-L130